app/models/redirection.rb

```diff
@@ -26,4 +26,14 @@ def tag_uri
 
     "tag:#{UNCHANGING_HOSTNAME},#{date}:#{url}"
   end
+
+  def unlink
+    previous_redirection = previous
+    next_redirection = self.next
```

Comment:
`next` is a keyword in Ruby, so it must be `self.next`. I've tried to think of something better that doesn't require the `self.`, but I like the `next`/`previous` names enough that I don't mind the `self.`.